spec/site_prism/page_objects/page/organisations/ucas_contacts.rb

```diff
@@ -2,17 +2,26 @@ module PageObjects
   module Page
     module Organisations
       class UcasContacts < PageObjects::Base
+        class ContactDetail < SitePrism::Section
+          element :heading, ".govuk-summary-list__value"
+          element :details, ".govuk-summary-list__value .govuk-body"
+          element :change_link, ".govuk-summary-list__actions .govuk-link"
```

Comment:
```suggestion
          element :change_link, ".govuk-summary-list__actions"
```

Shouldn't need to look for links inside actions section

Reply:
We click this so it needs to resolve to a link. I've change it to `".govuk-summary-list__actions a"`